src/components/common/LikelyToGiveSelect/LikelyToGiveSelect.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { LikelyToGiveEnum } from 'src/graphql/types.generated';
+import { LikelyToGiveSelect } from './LikelyToGiveSelect';
+
+jest.mock('src/utils/functions/getLocalizedLikelyToGive', () => ({
+  getLocalizedLikelyToGive: (_t: any, val: string) => `${val}`,
+}));
+
+describe('LikelyToGiveSelect', () => {
+  it('should render a Select with all LikelyToGiveEnum options', async () => {
+    render(<LikelyToGiveSelect value="" onChange={() => {}} />);
+
+    const select = screen.getByRole('combobox');
+    await userEvent.click(select);
+
+    Object.values(LikelyToGiveEnum).forEach((val) => {
```

Comment:
Usually, we try to put `expect`s in the main body of the test, instead of in an `if` bock or a loop. Instead of this being a loop, can you just test that one or two of the options are in the document?